### ISSUE TITLE
Timeline update?

### ISSUE THREAD

**Issue by josephrocca** [WebAssembly/memory64#36]
Wondering if anyone has an educated guess at when we'll get Memory64 unflagged in a stable version of Chromium or Firefox? Both currently have it available behind a flag, of course.

I [wasn't able to find](https://bugs.chromium.org/p/chromium/issues/list?q=component%3ABlink%3EJavaScript%3EWebAssembly&colspec=ID%20Pri%20Type%20Component%20Status%20Summary%20Owner%20Target%20M%20Reporter%20Modified%20Opened%20Stars&sort=-stars) any tracking bug for Memory64 in crbug. Maybe it's private? If anyone can point me in the right direction that would be handy.

Even very rough estimates (e.g. >1 year? >2 years?) would be useful!

**Related:**
 * https://github.com/WebAssembly/memory64/issues/20 - previously asked June 2021, Firefox had unknown ETA, and no word from any Chromium people.

**Comment by backes** [comment]
This is V8's (current) tracking bug: https://crbug.com/v8/13780
The tracking bug for the original implementation is here: https://crbug.com/v8/10949 (completed)

Memory64 is currently "staged" in V8, meaning it is enabled if you enable "Experimental WebAssembly" in the chrome flags: chrome://flags/#enable-experimental-webassembly-features

V8 will ship Memory64 once it reached stage 4, see https://v8.dev/docs/wasm-shipping-checklist.

**Comment by josephrocca** [comment]
Thanks @backes! Any rough estimate on how long it'll take to reach stage 4? A very rough estimate like "probably at least a year" or "*maybe* by the end of this year" is still useful for me.

**Comment by backes** [comment]
@sbc100 is the proposal owner. Not much is missing for phase 4, so it is likely to happen within Q2 or Q3. V8 will then ship shortly after that.

**Comment by alexDrinkwater** [comment]
@backes @sbc100 Any updates on this now that Q3 is almost over?

**Comment by dschuff** [comment]
As @backes mentioned in #42, V8 Chrome now has support that should be complete and working. Also @sbc100 has gotten emscripten into good shape (almost everything should work). I would really like to get some feedback from users that this works well for them before we go to phase 4. Perhaps you can help there! Beyond that, there needs to be one more implementation  (sccording to the roadmap, Firefox has one in Nightly), and I think @sbc100 said there was a little bit of specification-writing work to be done. But overall we are very close.

**Comment by sbc100** [comment]
The main blocker at this point is re-basing the test suite and adding the spec text.  Both of those are currently on my place but I would welcome any help with either of them.

**Comment by sbc100** [comment]
I created a tracking issue (#43) for folks who want to track the progress of this proposal.

Right now that main blockers are rebasing the interpreter changes (I'm working on that now), and updating the formal spec (this is also somewhat blocked on the rebase because there formal spec is also out-of-date with respect to upstream).

I'm hoping to we have both of these done by the new year.

Closing this issue now that we have #43
